README.md

```diff
@@ -3,13 +3,12 @@
 ## Usage
 
 Begin by declaring a type that defines the information you need to collect from the command line.
-Decorate each stored property with one of `ArgumentParser`'s property wrappers,
-and declare conformance to `ParsableCommand`.
+Decorate each stored property with one of `ArgumentParser`'s property wrappers.
 
 ```swift
 import ArgumentParser
 
-struct Repeat: ParsableCommand {
```

Comment:
I advise against this: it's important to know what the type is conforming to in its declaration. 

By moving it in the extension, people have to scroll to the extension to discover the conformation.

Reply:
But is this valid Swift though ? (If yes, then TIL)

I've seen other syntaxes that I know are valid yes, but never the case of declaring conformance without actually confirming on the declaration

```
protocol P {
  func pReq()
}

// Ok for that of course :
struct S1: P {
  func pReq()
}

// ok for that too;
struct S2 {}
extension S2: P {
  func pReq() {}
}

// And even also this:
struct S3 {
  func pReq()
}
extension S3: P {}

// But this is news to me though:
struct S4: P {}
extension S4 {
  func pReq()
}
```
If that's indeed valid Swift I'll be happy to revert and to end up learning something today, but it just felt surprising to me when I read that in the README in the first place that's all

Reply:
Yes, you it's valid 😃 here's a working example (you were missing the `{}` in the last extension):
```swift
protocol P {
  func pReq()
}

struct S4: P {}

extension S4 {
  func pReq() {}
}
```

Reply:
Wow, good to know, TIL 👍
I'm guessing the extension has to be in the same file? Or it also works if declared elsewhere (well as long as declared in the same module)?

Will revert those changes then 👍

Reply:
Yes, it works as long as it's defined in the same module (which means that having an extension in a separate file is also valid, as long as the extension is at least `internal`) 🚀

Reply:
I agree that it looks a bit weird with `run()` split out into an extension. What about replacing the closing bracket in the type declaration with a comment like `// continued below...` and then deleting the line that opens the extension? Then it would be just one type declaration that’s broken up by some explanatory text.